pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -445,6 +446,89 @@ var _ = Describe("Evacuation", func() {
 
 			controller.Execute()
 		})
+
+		It("Should not evict new VMIs when the amount of concurrent running migrations reached its maximum capacity", func() {
+			var maxParallelMigrationsPerCluster uint32 = 5
+			config, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{
+				MigrationConfiguration: &v1.MigrationConfiguration{
+					ParallelMigrationsPerCluster: &maxParallelMigrationsPerCluster,
+				},
+			})
+
+			controller = evacuation.
+				NewEvacuationController(
+					vmiInformer,
+					migrationInformer,
+					nodeInformer,
+					podInformer,
+					recorder,
+					virtClient,
+					config)
+
+			nodeName := "node01"
+			addNode(newNode(nodeName))
+
+			vmi1 := newVirtualMachineMarkedForEviction("testvm1", nodeName)
+			vmiFeeder.Add(vmi1)
+
+			for i := 1; i <= int(maxParallelMigrationsPerCluster); i++ {
+				migrationFeeder.Add(newMigration(fmt.Sprintf("mig%d", i), vmi1.Name, v1.MigrationRunning))
+			}
+
+			vmi2 := newVirtualMachineMarkedForEviction("testvm2", nodeName)
+			vmiFeeder.Add(vmi2)
+
+			controller.Execute()
+
+			migrationInterface.
+				EXPECT().
+				Create(gomock.Any(), &v13.CreateOptions{}).
+				Return(&v1.VirtualMachineInstanceMigration{ObjectMeta: v13.ObjectMeta{Name: "something"}}, nil).
+				Times(0)
+		})
+
+		It("Should create new evictions up to the configured maximum migrations per outbound node", func() {
+			var maxParallelMigrationsPerCluster uint32 = 10
+			var maxParallelMigrationsPerOutboundNode uint32 = 5
```

Comment:
nit: same as above - I would make these consts.
BTW, if the reason is that you have to use these as pointers below, I would suggest:
```go
MigrationConfiguration: &v1.MigrationConfiguration{
	ParallelMigrationsPerCluster:      pointer.Uint32(maxParallelMigrationsPerCluster),
	ParallelOutboundMigrationsPerNode: pointer.Uint32(maxParallelMigrationsPerOutboundNode),
},
```

Reply:
@iholder-redhat I would do so, the problem is that [current](https://github.com/kubevirt/kubevirt/blob/main/vendor/k8s.io/utils/pointer/pointer.go) `pointer` package in our `vendor` is outdated. We will have to bump it. I would suggest to do so in a follow-up PR, WDYT?

Reply:
Sounds good!